02-strings/strings.py

```diff
@@ -1,11 +1,18 @@
 def length(text):
+    '''Create a function length() that takes a string parameter. Return the length of that string.'''
```

Comment:
Should this be `implement` instead of create since the function is already defined?

Reply:
I used the same convention as `master` branch, which uses "create" consistently.